sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/directconnectivity/Uri.java

```diff
@@ -24,6 +24,10 @@ public class Uri {
     private volatile Instant lastUnhealthyPendingTimestamp;
     private volatile Instant lastTransitionToUnhealthyTimestamp;
     private volatile boolean isPrimary;
+    public static final String ATTEMPTING = "Attempting";
+    public static final String NOT_ATTEMPTING = "NotAttempting";
```

Comment:
Question for the native english speakers :-) - i would prefer "Ignoring" - but if "NotAttempting" better explains it is definitely ok as well.

Reply:
I also think ignoring is clearer.